File: 2_Sum_Of_Natural_Numbers/main.c
#include<stdio.h>

void Sum_of_N(int n)
{   
    int sum=0;
    for (int i=1 ; i<=n; i++)
    {
        sum = sum+i;
    }
    printf("%d", sum);
}

int main()
{
    int num;
    printf("Enter the number, till then you want the sum of N Natural Numbers: ");
    scanf("%d", &num);

    Sum_of_N(num);
    return 0;
}

File: 2_Sum_Of_Natural_Numbers/main.py
def Sum_Of_N(n):
    sum = 0
    for i in range(1,n+1):
        sum = sum + i
    print(sum)


if __name__ == "__main__":
    num = int(input("Enter the number, till then you want the sum of N Natural Numbers: "))
    Sum_Of_N(num)

